src/main/java/org/broadinstitute/hellbender/tools/spark/sv/discovery/inference/ImpreciseVariantDetector.java

```diff
@@ -20,13 +21,16 @@
     public static List<VariantContext> callImpreciseDeletionFromEvidenceLinks(final PairedStrandedIntervalTree<EvidenceTargetLink> evidenceTargetLinks,
                                                                               final ReadMetadata metadata,
                                                                               final ReferenceMultiSource reference,
-                                                                              final int impreciseEvidenceVariantCallingThreshold,
+                                                                              final int impreciseVariantEvidenceThreshold,
+                                                                              final int maxCallableImpreciseVariantDeletionSize,
                                                                               final Logger localLogger) {
         final List<VariantContext> impreciseVariants =
                 Utils.stream(evidenceTargetLinks)
                         .map(p -> p._2)
                         .filter(EvidenceTargetLink::isImpreciseDeletion)
-                        .filter(e -> e.getReadPairs() + e.getSplitReads() > impreciseEvidenceVariantCallingThreshold)
+                        // todo: for now, we set a limit on the size of deletion we can call solely based on imprecise evidence
```

Comment:
Can you create a ticket for this, just so that we don't forgot it like what happened in this z-score filtering?

Reply:
Added https://github.com/broadinstitute/gatk/issues/4372